### PR TITLE
cosmetic changes

### PR DESCRIPTION
**Pull request by herrriehm**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | yes
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| CHANGELOG.md  | no
| License       | MIT

- http to https
- replaced generic .com by specific TLD
- updated some dead urls

### CODE REVIEW

File: app/config/wallabag.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ wallabag_core:
             section: entry
         -
             name: diaspora_url
-            value: http://diasporapod.com
+            value: https://diasporafoundation.org/
```
Comment (tcitworld) [inline app/config/wallabag.yml]: Nope, this should be a Diaspora instance URL.

File: app/config/wallabag.yml [hunk 2]
```diff
@@ -64,11 +64,11 @@ wallabag_core:
             section: entry
         -
             name: shaarli_url
-            value: http://myshaarli.com
+            value: https://github.com/shaarli/Shaarli
```
Comment (tcitworld) [inline app/config/wallabag.yml]: Nope, this should be a Shaarli instance URL.

File: app/config/wallabag.yml [hunk 3]
```diff
@@ -64,11 +64,11 @@ wallabag_core:
...
             section: entry
         -
             name: scuttle_url
-            value: http://scuttle.org
+            value: https://github.com/scronide/scuttle
```
Comment (tcitworld) [inline app/config/wallabag.yml]: Nope, this should be a Scuttle instance URL.

File: app/config/wallabag.yml (hunk 1 above)
Comment (tcitworld) [reply app/config/wallabag.yml]: (It's only a placeholder URL for some real URL)

File: app/config/wallabag.yml (hunk 3 above)
Comment (herrriehm) [reply app/config/wallabag.yml]: please excuse this faulty edits

File: src/Wallabag/CoreBundle/Resources/translations/messages.es.yml [hunk 4]
```diff
@@ -264,7 +264,7 @@ entry:
         provided_by: 'Proporcionado por'
     new:
         page_title: 'Guardar un nuevo artículo'
-        placeholder: 'http://sitioweb.com'
+        placeholder: 'https://sitioweb.com'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.es.yml]: Why not .es?

**Comment by j0k3r** [comment]
Thanks @herrriehm

**Pull request merged**